SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -112,6 +112,19 @@ USpatialNetDriver::~USpatialNetDriver() = default;
 
 bool USpatialNetDriver::InitBase(bool bInitAsClient, FNetworkNotify* InNotify, const FURL& URL, bool bReuseAddressAndPort, FString& Error)
 {
+	if (!bConnectAsClient)
+	{
+		USpatialGameInstance* GameInstance = GetGameInstance();
+
+		if (GameInstance != nullptr)
+		{
+			if (GameInstance->GetSpatialWorkerType() == SpatialConstants::RoutingWorkerType)
+			{
+				NetServerMaxTickRate = 120;
```

Comment:
I wonder how heavy a UE instance for this purpose is (and what the cost to users is). Perhaps a future possibility is a tiny c worker that runs on the runtime node and the cost can be considered an extension of runtime provided functionality (and avoid interfering with bin-packing)

Reply:
Right now the impact has not been measured. The instance would likely be quite big since it would load the map as well, so that the first optim to consider.
Having a small C worker would likely be much better, but we do not have the infra to launch utility workers right now.